### PR TITLE
doc: New button and LED numbering follow-up PR

### PR DESCRIPTION
**Pull request by annwoj**
Follow up PR of the https://github.com/nrfconnect/sdk-nrf/pull/15756  because the previous one got merged prematurely.

The list of the reviewers pulled from the code owners of the previous PR.

Please double-check the (already) live documentation of the samples samples you're working on: 

[Coremark](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/benchmarks/coremark/README.html)
[Central and peripheral HR](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/central_and_peripheral_hr/README.html)
[Central BAS](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/central_bas/README.html)
[Central HIDS](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/central_hids/README.html)
[Fast pair input device](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/fast_pair/input_device/README.html)
[Fast pair locator tag](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/fast_pair/locator_tag/README.html)
[ISO time sync](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/iso_time_sync/README.html)
[Bluetooth mesh light ctrl](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/mesh/light_ctrl/README.html)
[Bluetooth mesh light dimmer](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/mesh/light_dimmer/README.html)
[Bluetooth mesh light switch](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/mesh/light_switch/README.html)
[Bluetooth mesh sensor client](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/mesh/sensor_client/README.html)
[Bluetooth mesh sensor server](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/mesh/sensor_server/README.html)
[Bluetooth mesh silvair enocean](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/mesh/silvair_enocean/README.html)
[Multiple adv sets](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/multiple_adv_sets/README.html)
[peripheral ams client](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_ams_client/README.html)
[Peripheral ANCS client](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_ancs_client/README.html)
[Peripheral BMS](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_bms/README.html)
[Peripheral CTS client](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_cts_client/README.html)
[Peripheral GATT DM](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_gatt_dm/README.html)
[Peripheral HIDS keyboard](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_hids_keyboard/README.html)
[Peripheral HIDS mouse](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_hids_mouse/README.html)
[Peripheral HR coded](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_hr_coded/README.html)
[Peripheral LBS](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_lbs/README.html)
[NFC pairing](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_nfc_pairing/README.html)
[Peripheral RSCS](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_rscs/README.html)
[Peripheral status](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_status/README.html)
[Peripheral UART](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_uart/README.html)
[Bluetooth throughput](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/throughput/README.html)
[ESB receiver](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/esb/esb_prx/README.html)
[ESB transmitter](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/esb/esb_ptx/README.html)
[NFC record launch app](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nfc/record_launch_app/README.html)
[NFC record text](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nfc/record_text/README.html)
[NFC shell](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nfc/shell/README.html)
[NFC system off](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nfc/system_off/README.html)
[NFC Tnep_tag](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nfc/tnep_tag/README.html)
[NFC writable ndef msg](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nfc/writable_ndef_msg/README.html)
[Peripheral 802154_phy_test](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/peripheral/802154_phy_test/README.html)
[SUIT smp transfer](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/suit/smp_transfer/README.html)

### CODE REVIEW

**Review by peknis — APPROVED** [review]

**Review by MarekPieta — APPROVED** [review]

File: samples/peripheral/802154_phy_test/README.rst [hunk 1]
```diff
@@ -951,7 +951,7 @@ After programming the sample to your development kit, complete the following ste
 
 #. To turn the LEDs on, set the respective pin's state to low to let the current flow through the LED, using the ``custom lsetgpio <pin> 0`` command, where ``<pin>`` is the number of the pin assigned for selected LED.
 
-   See the following example for how to light up LED 1 on the nRF5340DK:
+   See the following example for how to light up **LED 1** on the nRF5340DK:
```
Comment (umapraseeda) [inline samples/peripheral/802154_phy_test/README.rst:954]:
```suggestion
   See the following example for how to light up **LED 1** on the nRF5340 DK:
```

**Comment by divipillai** [comment]
Some suggestions: 
* For the [ISO time sync](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/iso_time_sync/README.html) sample , sample structure, it feels like the source files under the tab are part of the "
iso_rx.c" source file, which is not the case. It would be better to add all the source file under the tab.

* [Peripheral HIDS mouse](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_hids_mouse/README.html) - Testing with nRF connect for desktop - shows both tab as nRF52 and nrf53 DKs - https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_hids_mouse/README.html#testing-with-nrf-connect-for-desktop.

* Since nRF54L15 is considered as PDK should this be reflected in the tab name, which mentions nRF54 DKs?

**Comment by annwoj** [comment]
> Some suggestions:
> 
> * For the [ISO time sync](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/iso_time_sync/README.html) sample , sample structure, it feels like the source files under the tab are part of the "
>   iso_rx.c" source file, which is not the case. It would be better to add all the source file under the tab.
> * [Peripheral HIDS mouse](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_hids_mouse/README.html) - Testing with nRF connect for desktop - shows both tab as nRF52 and nrf53 DKs - https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/peripheral_hids_mouse/README.html#testing-with-nrf-connect-for-desktop.
> * Since nRF54L15 is considered as PDK should this be reflected in the tab name, which mentions nRF54 DKs?

Good catch - thank you Divya! Will apply. 
As for the PDK part, as far as I'm aware the agreement is to keep it simplified, since similarly to 54H, the PDK will be changed to DK. If we introduce PDK now, soon we will have to revert all the changes.

**Review by alstrzebonski — APPROVED** [review]
Fast Pair approved

**Review by divipillai — APPROVED** [review]

**Review by tomchy — APPROVED** [review]

**Pull request merged**